addons/project/static/src/js/components/project_utils.js

```diff
@@ -0,0 +1,165 @@
+/** @odoo-module **/
+
+import { _lt } from 'web.core';
+import { ComponentAdapter } from 'web.OwlCompatibility';
+import { FormViewDialog } from 'web.view_dialogs';
+const { useState, useRef } = owl.hooks;
+
+export class FormViewDialogComponentAdapter extends ComponentAdapter {
+    renderWidget() {
+        // Ensure the dialog is properly reconstructed. Without this line, it is
+        // impossible to open the dialog again after having it closed a first
+        // time, because the DOM of the dialog has disappeared.
+        return this.willStart();
+    }
+}
+
+const components = {
+    FormViewDialogComponentAdapter,
+};
+
+class MilestoneComponent extends owl.Component {
```

Comment:
I would say no, because we only use it in this file.

As it's not a default component (like we would have in web) I think that if one needs to use it in the future, he/she may change the code and export it ? (Once again arbitrary choice.)

Reply:
Yeah, that's the idea, but for the moment, you can let as you have done, we will add the export if needed. 🙂